xwf/gateway/deploy/roles/fbstatsd/tasks/main.yml

```diff
@@ -13,12 +13,19 @@
     dest: /etc/hosts
     mode: '0644'
 
-- name: install the fb yum repo file
-  template:
-    src: xwf-s3.repo.j2
-    dest: /etc/yum.repos.d/xwf-s3.repo
-    mode: '0440'
+- name: install the cob yum repo using bootstrapper  script
+  shell: |
+   curl -L "https://www.facebook.com/xwf/bootstrap/?access_token={{ AAA_ACCESS_TOKEN|replace('\n', '')|trim }}&entity=xwf.{{ partner }}.qa.radius.999" > /tmp/cob_test.sh
```

Comment:
this script is unsafe and will move all our other repos we use in yum for magma use.

Reply:
I don't think that this is the best approach. 
I gave the bootstrap script as an example on how to use, but as you only need the credentials, you can just curl the `machine_config` endpoint: 

Taken from the task comments (T84953710)
```
curl https://graph.expresswifi.com/machine_config?access_token=<TOKEN> | jq .
{
  "data": [
    {
      "yum_s3_plugin_config": {
        "access_key": "ASIAQS7OT3XNAKDFGO4D",
        "secret_key": "XXXX",
        "session_token": "XXXXXXXXXXXX"
      }
    }
  ]
}
```